bindings/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
* As commented above, I'd move this to another repo.
* No need for initial empty line